man/systemd.network.xml

```diff
@@ -1188,6 +1188,16 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>UserClass=</varname></term>
+          <listitem>
+            <para>A DHCPv4 client can use UserClass option to identify the type or category of user or applications
+            it represents. The information contained in this option is an string object that represents the user class
+            of which the client is a member. Each class sets an identifying string of information to be used by the DHCP
+            service to classify identifying clients. Takes a whitespace-separated list of user class information.</para>
+          </listitem>
```

Comment:
Second phrase should be "**a** string object", not "**an** string object". Maybe I'd also start with the
> A whitespace-separated list of…

like a lot of other options.